Gosu/Bitmap.hpp

```diff
@@ -31,6 +31,11 @@ namespace Gosu
         : w(w), h(h), pixels(w * h, c)
         {
         }
+
+        Bitmap(unsigned w, unsigned h, std::vector<Color> pixels)
+        : w(w), h(h), pixels(pixels)
```

Comment:
Should call std::move on pixels to prevent a copy

Reply:
But it also doesn't look like you're using this new constructor :)

Reply:
Ah, just saw your comment -- n/m

Reply:
Nope. Not any more. Tried different things with memcpy the data directly, which didn't worked. Then I tried this solution by assigning the pixels vector directly instead of using set_pixel, which didn't worked either...